pkgs/build-support/build-fhs-userenv/default.nix

```diff
@@ -1,8 +1,9 @@
-{ runCommand, lib, writeText, writeScriptBin, stdenv, ruby } :
-{ env, runScript ? "bash", extraBindMounts ? [], extraInstallCommands ? "" } :
+{ runCommand, lib, writeText, writeScriptBin, stdenv, bash, ruby } :
+{ env, runScript ? "${bash}/bin/bash", extraBindMounts ? [], extraInstallCommands ? "" } :
```

Comment:
This one could be left and that way the user would get whatever bash is in their path as the default which could be considered a feature. I think it is best like this as it won't change depending on how you call it but either way makes sense.